components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-authentication-flow.jsp

```diff
@@ -358,26 +344,66 @@ var img = "";
 			}
         });
 
-        var templates = $.parseJSON('<%=templatesJson%>');
+        populateTemplates();
+		function populateTemplates(){
```

Comment:
formatting issue, probably due to tab character

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/1570/commits/e914bbda0f0f9d89d1bb6480bb19448ff6b94485